driver/csiplugin/controllerserver.go

```diff
@@ -278,6 +278,18 @@ func (cs *ScaleControllerServer) setQuota(ctx context.Context, scVol *scaleVolum
 		}
 	}
 
+	// changing volsize here for pvc size in decimal units to align with scale block size
+	filesystemname := scVol.VolBackendFs
+	klog.Info("Filesystemname", filesystemname)
+	filesystemdetails, err := cs.Driver.connmap["primary"].GetFilesystemDetails(ctx, filesystemname)
```

Comment:
the connector depends upon the volBackendFs and the respective scale connector. and scVol contains the same. 
```suggestion
	filesystemdetails, err := scVol.Connector.GetFilesystemDetails(ctx, filesystemname)
```